docs/source/admin/quick_howto/ciab.rst

```diff
@@ -99,6 +99,68 @@ variables.env
 .. [2] Consider ``make -j`` to build quickly, if your computer can handle multiple builds at once.
 .. [3] Please do NOT use the Perl endpoints directly. The CDN will only work properly if everything hits the Go API, which will proxy to the Perl endpoints as needed.
 
+X.509 SSL/TLS Certificates
+=========================
+All components in Apache Traffic Control utilize SSL/TLS secure communications by default.  For SSL/TLS connections to properly validate within the "CDN in a Box" container network a shared self-signed X.509 Certificate Authority (CA) is generated at the first initial startup.  Additional self-signed wildcard certificates are generated for each container service and all delivery services of the CDN.  All certificates and keys are stored in the ``ca`` host volume which is located at ``infrastruture/cdn-in-a-box/traffic_ops/ca`` [4]_. 
+
+.. _ciab-x509-certificate-list:
+.. table:: Self-Signed X.509 Certificate List
+
+   +---------------------------+-----------------------------------+------------------------------+
+   | Filename                  | Description                       | X.509 CN/SAN                 |                  
+   +===========================+===================================+==============================+
+   | CIAB-CA.crt               | Shared CA Certificate             | N/A                          |
+   +---------------------------+-----------------------------------+------------------------------+
+   | infra.ciab.test.crt       | Infrastruture Certificate         | \*.infra.ciab.test           |
+   +---------------------------+-----------------------------------+------------------------------+
+   | demo1.mycdn.ciab.test.crt | Demo1 Delivery Service Certificate| \*.demo1.mycdn.ciab.test     |
+   +---------------------------+-----------------------------------+------------------------------+
+   | demo2.mycdn.ciab.test.crt | Demo2 Delivery Service Certificate| \*.demo2.mycdn.ciab.test     |
+   +---------------------------+-----------------------------------+------------------------------+
+   | demo3.mycdn.ciab.test.crt | Demo3 Delivery Service Certificate| \*.demo3.mycdn.ciab.test     |
+   +---------------------------+-----------------------------------+------------------------------+
```

Comment:
This indentation is inconsistent - above this point tabs were used. Can you either change the existing indentation to spaces or change this to tabs? I guess it doesn't matter which, as long as it's consistent, but for the record I think tabs are the superior choice.

Reply:
+1 on keeping existing indentation and making this consistent with it

Reply:
Sounds good.  I'll update my .vimrc to not replace tabs with spaces.